cdk_build/DigitFtpStack.py

```diff
@@ -67,3 +68,32 @@ def __init__(self, scope: Construct, construct_id='digitNbinTransfer', **kwargs)
         rule = self.cron(aws_events, self.cron_schedule,)
         rule.add_target(aws_events_targets.LambdaFunction(
             digit_nbin_transfer, retry_attempts=10))
+
+    # topic = sns.Topic(self, "Topic")
```

Comment:
Note this may need to be a distinct lambda, and may need to add permissions to publish sns topics